onnxscript/function_libs/torch_lib/_flags.py

```diff
@@ -0,0 +1,11 @@
+"""Experimental flags.
+
+NOTE: These flags are experimental only. Any flag here can be removed at any
```

Comment:
Should we emit warning whenever flag uses under this file is detected?

Reply:
Good idea

Reply:
And raise error when this flag is removed but still used by user.

Reply:
Done